src/user/model.coffee

```diff
@@ -91,12 +91,8 @@ User =
         User.channel.emit('change')
 
   destroy: ->
-    User.channel.removeAllListeners()
-
```

Comment:
@pandafulmanda I think this is the culprit for why the 403 on user change is still occurring.  When I put a breakpoint here, it appears that webview is calling `destroy()` on every navigation, which then removes `task/collection` and `exercise/collection` listeners